File: Clase1/Lab02/20222828_Lab_2 (1)/20222828_Lab_2/Question.java
import java.util.List;

abstract class Question implements Printable{
	protected String prompt;
	protected int code;
	protected List<String> options;

	//Constructor
	public Question(int code, String prompt, List<String> options){
		this.code = code;
		this.prompt = prompt;
		this.options = options;
	}

	public void setPrompt(String prompt){
		this.prompt = prompt;
	}

	public String getPrompt(){
		return this.prompt;
	}

	public void setCodPregunta(int codPregunta){
		this.code = codPregunta;
	}

	public int getCodPregunta(){
		return this.code;
	}

	public List<String> getOptions() { return options; }
}

File: Clase6_Lab4/Laboratorio4-Fuentes-C# (1)/Laboratorio4-Fuentes-C#/AssessmentConsoleApp/Question.cs
public abstract class Question : IPrintable
{
    private int code;
    protected string prompt;

    public Question(int code, string prompt)
    {
        this.code = code;
        this.prompt = prompt;
    }

    public int Code
    {
        get { return code; }
    }

    public string Prompt
    {
        get { return prompt; }
    }

    public abstract string DevolverDatos();
}